FEXCore/Source/Interface/Core/OpcodeDispatcher.h

```diff
@@ -46,6 +46,12 @@ enum class BTAction {
   BTComplement,
 };
 
+enum class ForceTSOMode {
+  Default,
```

Comment:
This seems to be a good place to mention what the default is.

Reply:
I think this value needs renaming, since I actually misunderstood what "Default" means here. IIUC, this value is actually used for unannotated regions, for which should FEX's normal TSO configuration should be used. If that's correct, maybe something like "NoOverride" is a clearer name?